### PR TITLE
Automated cherry pick of #84: Install knative-serving into its own namespace. #87: Need to create the knative-serving namespace in the proper Cherry pick of #84 #87 on v1.1-branch. #84: Install knative-serving into its own namespace. #87: Need to create the knative-serving namespace in the proper

### PR DESCRIPTION
**Pull request by jlewi**
Related to #74

### CODE REVIEW

**Comment by jlewi** [comment]
/assign @Bobgy 

@Bobgy This is ready for review. I will remove the hold after verifying that it is working on master.

/hold

**Comment by Bobgy** [comment]
Would you mind cleaning up the PR's title?
That was the issue I mentioned when using the automated PR cherrypick tool.

/lgtm

File: kubeflow/Makefile [hunk 1]
```diff
@@ -188,7 +192,7 @@ apply-asm: hydrate
 	# TODO(jlewi): Should we use the newer version in asm/asm
 	# istioctl manifest --context=${KFCTXT} apply -f ./manifests/gcp/v2/asm/istio-operator.yaml
 	# TODO(jlewi): Switch to anthoscli once it supports generating manifests
-	# anthoscli apply -f ./manifests/gcp/v2/asm/istio-operator.yaml
+	# anthoscli apply -f ./manifests/gcp/v2/asm/istio-operator.yaml
```
Comment (Bobgy) [inline kubeflow/Makefile:195]: nit: I typically enable vscode auto formatting to get rid of unwanted trailing whitespaces.

File: kubeflow/Makefile [hunk 2]
```diff
@@ -198,7 +202,12 @@ apply-kubeflow: hydrate
 	kubectl --context=$(KFCTXT) apply -f ./$(BUILD_DIR)/metacontroller
 	kubectl --context=$(KFCTXT) apply -f ./$(BUILD_DIR)/application
 	kubectl --context=$(KFCTXT) apply -f ./$(BUILD_DIR)/cloud-endpoints
-	kubectl --context=$(KFCTXT) apply -f ./$(BUILD_DIR)/iap-ingress
+	kubectl --context=$(KFCTXT) apply -f ./$(BUILD_DIR)/iap-ingress	
+
+	# Apply the namespace first
+	kubectl --context=${KFCTXT} apply -f ./$(BUILD_DIR)/knative/~g_v1_namespace_knative-serving.yaml
```
Comment (Bobgy) [inline kubeflow/Makefile:208]: I thought `kubectl` could apply namespace first, at least when I was using `kubectl apply -k` on a kustomize target. Shall we change the command to pipe kustomize output into kubectl?

**Comment by Bobgy** [comment]
/approve
let's get this in anyway first to test auto deploys

**Comment by Bobgy** [comment]
How did you verify it's working on master?

**Comment by jlewi** [comment]
@Bobgy I used the autodeployments
https://kf-vbp-0721-cff.endpoints.kubeflow-ci-deployment.cloud.goog/

I just checked the namespace knative-serving wasn't empty.

The test appears to be still having problems but I don't want to keep blocking the release on that.

**Pull request merged**